### ISSUE TITLE
Port Clap Preset Discovery scan enhancements to the PAtchDB scanner

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#7463]
once preset discovery is done

**Comment by baconpaul** [comment]
I opened this as a separate issue since I want it in a separate commit for bisect purposes.

**Comment by baconpaul** [comment]
actually i'll bundle it all together.
